components/insight/SRC/org/openmicroscopy/shoola/util/ui/lens/LensModel.java

```diff
@@ -303,6 +303,7 @@ BufferedImage getZoomedImage()
 																		null);
 			return scaleBufferedImage(img, zoomFactor, zoomFactor);
 		} catch (Exception e) {
+		    e.printStackTrace();
```

Comment:
Kind of... The class is not part of an agent package, so I can't get a logger via an agents registry. Is there another method to get access to a 'general' Insight logging instance @jburel ?